addon/styles/addon.scss

```diff
@@ -39,19 +42,25 @@
   align-items: flex-end;
 }
 
+
 .frost-file-picker .frost-text,
 .frost-file-picker .file-select-overlay .text {
+  transition: border-bottom-color 1s ease;
   width: 100%;
 }
 
 .frost-file-picker .frost-text:read-only {
-  border-bottom: 1px solid $frost-color-grey-6;
+  border-bottom-color: $frost-color-grey-6;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
```

Comment:
Then please add a comment in the CSS to reflect this.
```sass
// Note: using separate border properties instead of "border-bottom" due to issues with animations/transitions
border-bottom-color: $frost-color-grey-6;
border-bottom-style: solid;
border-bottom-width: 1px;
```